pkg/gitserver/gitserver.go

```diff
@@ -60,6 +62,14 @@ GIT_INITIAL_CLONE_*=<url>[;<name>]
   each environment variable in this pattern will be cloned when the process starts; failures will be logged
   <name> must be [A-Z0-9_\-\.], the cloned directory name will be lowercased. If the name is invalid the
   process will halt. If the repository already exists on disk, it will be updated from the remote.
+BUILDCONFIG_AUTOSTART=yes|no
+  if 'yes', builds will be started automatically for BuildConfigs that have a generic webhook defined
+  and are configured with source on this server.
```

Comment:
also true/false is more standard i think (though i admit less "englishy" in this case)